server/views/components/content-tile-featured/template.njk

```diff
@@ -19,9 +19,9 @@
     <p class="content-link--external">{{ params.displayUrl }}</p>
   {% elif params.contentType === "internal_link" %}
     <p class="content-link--internal"></p>
-  {% elif params.contentType === "series" or params.contentType === "category" %}
-    <p class="content-link--collection">Collection</p>
   {% else %}
-    <p class="content-link--article">Read</p>
+    {% if params.contentType !== "series" and params.contentType !== "category" %}
```

Comment:
Could just put the `if` inside the `else` as an `elif` ?

Reply:
your rite, updating.